Dockerfile

```diff
@@ -247,13 +247,19 @@ RUN /bin/bash -c 'phpbrew init && source ~/.phpbrew/bashrc && phpbrew install 5.
 
 ################################################################################
 #
-# Emacs Cask
+# Emacs and Cask
 #
 ################################################################################
 USER root
-RUN add-apt-repository ppa:kelleyk/emacs
-RUN apt-get -y update
-RUN apt-get -y install emacs25-nox
+
+RUN wget http://ftpmirror.gnu.org/emacs/emacs-25.2.tar.gz && \
```

Comment:
@Benaiah Looks like PPA is no longer used in the updated PR.

Reply:
Sorry, so understood.. you don't have any issue with the updated PR, and we are waiting on feedback from @calavera. 

@calavera Please look at this PR.